PSICover/PSICover.cs

```diff
@@ -160,21 +162,70 @@ class Analyzer {
                blocks.RemoveAt (i - 1);
          blocks.Reverse ();
 
+         int hitscount = 0;

```

Comment:
Coding convention for count is variable should start with small letter c so we should use cHits

Reply:
Also, we should declare variable right before it is used. In this case, it should be in line 169

Reply:
updated

![image](https://github.com/SasiTMM/PSI/assets/125174306/56c993a1-7af1-439d-85af-bb4dfec1b570)